Example/TABTestKit_ExampleUITests/CollectionViewScreen.swift

```diff
@@ -17,3 +17,11 @@ struct CollectionViewScreen: ScrollableScreen {
   lazy var lastCell = trait.cell(matchingID: "CollectionCell 49")
   
 }
+
+extension CollectionViewScreen: Refreshable {
+    
+  func refresh() {
+    // Pull the collection view down to refresh
+    trait.scroll(.upwards)
```

Comment:
Do we need to make sure that you're at the top of the screen? There could be a chance that we're in a scrolled position and this won't work. Same for the other one below